github-merge-flow.jsonc

```diff
@@ -15,6 +15,11 @@
         "release/8.0-rc":{
             "MergeToBranch": "release/8.0",
             "ExtraSwitches": "-QuietComments"
+        },
+        // Automate merging runtime release/9.0-rc branches back to release/9.0
+        "release/9.0-rc":{
```

Comment:
I don't think are wildcarded like they were in old Maestro. Split into an rc1 and rc2 entry.